src/pages/documentation/faq.md

```diff
@@ -84,20 +84,36 @@ iOS
-
-* Mobile Core, which includes the Lifecycle, the Identity, and the Signals extensions, is required for all other extensions. The final app size increase can be calculated by adding the Mobile Core size to each of the enabled extensions. For example, the iOS app distribution using the Target and Analytics extensions will have a total size increase of 635 KB. (Core: 504 KB + Analytics: 54 KB + Target: 77 KB).
-* The iOS (SDK extension versions 2+) estimates are based on Xcode’s App Thinning size report for one architecture. The Android (SDK extension versions 1+) size estimates listed refer to unsigned apps and do not account for proguarding.
+#### Adobe Experience Platform SDKs
+| Extension                 | iOS (KB) | Android (KB) | Additional Dependencies†                          |
+| :------------------------ | :------- | :----------- | :------------------------------------------------ |
+| Core                      | 925      | 497          | Android only: `kotlin-stdlib`                     |
+| Identity                  | 136      | 51           |                                                   |
+| Lifecycle                 | 150      | 59           |                                                   |
+| Signal                    | 18       | 33           |                                                   |
+| Edge Network extension    | 253      | 99           | Identity for Edge Network                         |
+| Consent for Edge Network  | 33       | 25           | Edge Network extension, Identity for Edge Network |
+| Identity for Edge Network | 69       | 49           |                                                   |
+| Analytics                 | 133      | 95           |                                                   |
+| Assurance                 | 917      | 527          | Android only: `androidx.constraintlayout`         |
+| Audience                  | 82       | 44           | Identity                                          |
+| Campaign                  | 166      | 80           | Identity                                          |
+| Campaign Classic          | 34       | 42           | Identity                                          |
+| Edge Bridge               | 0.84     | 6            |                                                   |
+| Media                     | 283      | 137          |                                                   |
+| Messaging                 | 33       | 77           | Edge Network extension, Identity for Edge Network |
+| Optimize                  | 88       | 80           | Edge Network extension, Identity for Edge Network |
+| Places                    | 799      | 59           |                                                   |
+| Target                    | 283      | 99           |                                                   |
+| User Profile              | 18       | 29           |                                                   |
```

Comment:
Just as a heads up, you don't actually need all the whitespace between the text and the next line, since Markdown will automatically convert it to be fine ;)